src/NuGetPackageVerifier/PackageIssueFactory.cs

```diff
@@ -8,6 +8,17 @@ namespace NuGetPackageVerifier
 {
     public static class PackageIssueFactory
     {
+        public static PackageVerifierIssue AssemblyMissingAuthorAttribute(string assemblyPath)
+        {
+            return new PackageVerifierIssue(
+                "AUTHOR_ATTRIBUTE",
+                assemblyPath,
+                string.Format(
+                    @"The managed assembly '{0}' in this package is missing the '[assembly: AssemblyMetadata(""Author"", ""<text>"")]' attribute.",
```

Comment:
Plus, this is ultimately about the *package* having a wrong/missing author, not the assembly.